catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -519,7 +519,7 @@ public Response getDocument(@PathParam("sourceid") String sourceid, @PathParam("
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.INTERNAL_SERVER_ERROR);
             } catch (CatalogTransformerException e) {
-                String exceptionMessage = "Unable to transform Metacard.  Try different transformer: ";
+                String exceptionMessage = "Unable to transform Metacard : \n" + e.getMessage();
```

Comment:
We want a way to convey error messages to the client via the rest endpoint without exposing implementation details. Wondering if @pklinef / @stustison / @andrewkfiedler can weigh in on this?